src/services/comments.js

```diff
@@ -88,6 +79,13 @@ module.exports = class CommentsHelper {
 				result: updatedComment,
 			})
 		} catch (error) {
+			if (error.message == 'COMMENT_INVALID' || error.message == 'COMMENT_PARENT_INVALID') {
```

Comment:
@adithyadinesh0412  
 It should default to error.message || error in case any other error is thrown from the database or if the  message isn't defined inside error.